spec/order_spec.rb

```diff
@@ -0,0 +1,39 @@
+require 'order'
+
+describe Order do
+  let(:menu) { double :menu, list: "juice - £1.00" }
+  let(:menu_class) { double :menu_class, new: menu }
```

Comment:
Great use of mocking here! 

Curious as to why you are passing in the reference to the class? I think in this case passing in the object itself should be ok since the Order object only needs to refer to one Menu object

Reply:
Thank you for your comments, Lindsey! I did actually do the dependency injection challenge during the week before the Takeaway challenge, and was relying on it when creating doubles (hence both class and object doubles as I was just following what was suggested there), but clearly I have misunderstood it :) (am working on resolving the misunderstanding...)